syntaxes/wdl.tmGrammar.json

```
@@ -96,7 +96,10 @@
       "include": "#input-block"
     },
     {
-      "include": "#command-block"
+      "include": "#command-block-angle"
```

Nit: when I think "angle bracket", I think `<` and not `{` (a "brace" or "curly bracket").

```suggestion
      "include": "#command-block-brace"
```

---

I'm going to go with curly to match the `punctuation.bracket.curly` names above.